tests/phpunit/tests/admin/wpAutomaticUpdater.php

```diff
@@ -0,0 +1,572 @@
+<?php
+
+/**
+ * @group upgrade
+ *
+ * @covers WP_Automatic_Updater
+ */
+class Tests_Admin_WpAutomaticUpdater extends WP_UnitTestCase {
+	/**
+	 * An instance of WP_Automatic_Updater.
+	 *
+	 * @var WP_Automatic_Updater
+	 */
+	private static $updater;
+
+	/**
+	 * WP_Automatic_Updater::send_plugin_theme_email
+	 * made accessible.
+	 *
+	 * @var ReflectionMethod
+	 */
+	private static $send_plugin_theme_email;
+
+	/**
+	 * Sets up shared fixtures.
+	 */
+	public static function set_up_before_class() {
+		require_once ABSPATH . 'wp-admin/includes/class-wp-automatic-updater.php';
+		self::$updater = new WP_Automatic_Updater();
+
+		self::$send_plugin_theme_email = new ReflectionMethod( self::$updater, 'send_plugin_theme_email' );
+		self::$send_plugin_theme_email->setAccessible( true );
+
+		add_filter( 'pre_wp_mail', '__return_false' );
```

Comment:
This will need to go in `set_up()` as hooks are cleared as part of the default tear down

https://github.com/WordPress/wordpress-develop/blob/13e076e6a48deb388bcba3a23beea2a2465a82d5/tests/phpunit/includes/abstract-testcase.php#L194

Reply:
Ah, my bad. I thought the hooks were also cleared on class tear down.